test/integration/model/nested_include/top_camera/top_camera.sdf

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" ?>
+<sdf version="1.7">
+  <model name='top_camera'>
+
+    <link name="top_camera_link">
+      <visual name="camera_visual">
+        <pose>-0.2 0 0.3 0 0 3.14</pose>
+        <geometry>
+          <box>
+            <size>0.1 0.1 0.1</size>
+          </box>
+        </geometry>
+        <!-- <geometry>
+          <sphere>
+            <radius>0.05</radius>
+          </sphere>
+        </geometry> -->
```

Comment:
this shows the new <geometry> block that is going to be the replacement of the current one? maybe add a comment?

Reply:
a9238215ab6ecae86f1d5b93aa72e75fdfec977b